src/PostgREST/SchemaCache.hs

```diff
@@ -317,6 +318,12 @@ decodeFuncs =
                       | v == 's' = Stable
                       | otherwise = Volatile -- only 'v' can happen here
 
+    parseFuncSetting :: Text -> FuncSetting
+    parseFuncSetting txt = toTuple $ T.splitOn "=" txt
+      where
+        toTuple [x,y] = (x,y)
+        toTuple _     = ("error","parsing")
+
```

Comment:
The decoding would then need to use `compositeArrayColumn`, like done here:

https://github.com/PostgREST/postgrest/blob/49584728b6423131712518e2f0803347c80069a1/src/PostgREST/Config/Database.hs#L181-L182

This won't need text splitting on the Haskell code.